depends/builders/openbsd.mk

```diff
@@ -1,7 +1,7 @@
 build_openbsd_CC = clang
 build_openbsd_CXX = clang++
 
-build_openbsd_SHA256SUM = sha256
+build_openbsd_SHA256SUM = sha256 -r
```

Comment:
I believe the same applies to [FreeBSD](https://man.freebsd.org/cgi/man.cgi?query=sha256) as well.

Reply:
I'll spin up a FreeBSD 14.1 VM to try that.